.github/workflows/ci.yml

```diff
@@ -7,43 +7,96 @@ on:
     branches: [master]
 
 jobs:
-  build:
-    strategy:
-      matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest ]
-    runs-on: ${{ matrix.os }}
+  initialize-ubuntu:
+    runs-on: ubuntu-latest
+    env:
+      SCHEME: scheme
     steps:
-      - uses: actions/checkout@v2
-
-      - uses: purescript-contrib/setup-purescript@main
-        with:
-          purescript: "0.15.8"
-          purs-tidy: "0.9.2"
-          psa: "0.8.2"
-          spago: "0.20.9"
-
-      - name: Cache PureScript dependencies
-        uses: actions/cache@v2
-        with:
-          key: ${{ runner.os }}-spago-${{ hashFiles('**/*.dhall') }}
-          path: |
-            .spago
-            output
-
-      - name: Set up Node toolchain
-        uses: actions/setup-node@v2
-        with:
-          node-version: "16"
+      - name: Install chez scheme
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y chezscheme
```

Comment:
One issue that I'm noticing is that the Chez Scheme versions installed by each OS is slightly different:
- Linux: 9.5.4
- Mac 9.5.8
- Windows: 9.5.9

I assume we want to standardize on a single version across CI. Thoughts?